CHANGELOG.md

```diff
@@ -60,6 +60,9 @@ All notable changes to this project will be documented in this file.
 * ### NI-FGEN
     * #### Added
     * #### Changed
+        * Some functions missed setting repeated capabilities, leaving these as parameters instead of using the repeated capabilites object.
+            * `session.configure_digital_edge_script_trigger('0', source, ...)` becomes `session.script_triggers[0].configure_digital_edge_script_trigger(source, ...)`
+            * `session.configure_digital_level_script_trigger('0', source, ...)` becomes `session.script_triggers[0].configure_digital_level_script_trigger(source, ...)`
```

Comment:
The "before" example is wrong. It should be  `session.configure_digital_level_script_trigger('ScriptTrigger0', source, ...)`

Reply:
Fixed